.github/workflows/deploy.yml

```
@@ -0,0 +1,38 @@
+# yamllint --format github .github/workflows/test.yml
+---
+name: Deploy master and release versions
+
+# We deploy on master and release versions, regardless of if the commit is
+# documentation-only or not.
+on:
+  push:
+    tags: '[0-9]+.[0-9]+.[0-9]+'
+    branches: master
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04  # newest available distribution, aka focal
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          # license-maven-plugin needs the full history to generate copyright
+          # year range. Ex. 2013-2020. Don't do a shallow clone!
+          fetch-depth: 0
+      - name: Cache docker
+        uses: actions/cache@v2
+        with:
+          path: ~/.docker
+          key: ${{ runner.os }}-docker-${{ hashFiles('**/Dockerfile') }}
+          restore-keys: ${{ runner.os }}-docker
+      - name: Configure Deploy
+        run: build-bin/configure_deploy
+        with:
+          # GH_USER=user_that_created_GH_TOKEN
```

```suggestion
          # GH_USER=<user that created GH_TOKEN>
```

Just got a bit confused since it looked like code